docs/integrations/sources/github.md

```diff
@@ -35,7 +35,7 @@ Log into [GitHub](https://github.com) and then generate a [personal access token
 3. On the source setup page, select **GitHub** from the Source type dropdown and enter a name for this connector.
 4. Click `Authenticate your GitHub account` by selecting Oauth or Personal Access Token for Authentication. 
 5. Log in and Authorize to the GitHub account.
-6. **Start date** - The date from which you'd like to replicate data for streams: `comments`, `commit_comment_reactions`, `commit_comments`, `commits`, `deployments`, `events`, `issue_comment_reactions`, `issue_events`, `issue_milestones`, `issue_reactions`, `issues`, `project_cards`, `project_columns`, `projects`, `pull_request_comment_reactions`, `pull_requests`, `pull_requeststats`, `releases`, `review_comments`, `reviews`, `stargazers`, `workflow_runs`, `workflows`.
```

Comment:
issue milestones - can be filtered by "start_date" option

Reply:
The `since` param is not documented and [doesn't work ](https://docs.google.com/spreadsheets/d/1cjl_DZYAJVwTupbV6xCb3GpecEezf6R7Shijw0S23NI/edit?usp=sharing#gid=1846245387)for IssuesMilestones. My understanding is that if this was the case, we could make it fully incremental.

```bash
curl \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer TOKEN" \
  "https://api.github.com/repos/airbytehq/airbyte/milestones?state=all&since=2022-11-01T00:00:00Z&per_page=100" | jq -r '.[] | [.id, .due_on // "YYYY-MM-DDTHH:MM:SSZ", .created_at, .updated_at, .closed_at] | @tsv'
```

Reply:
issue_milestones - is semi incremental stream, it means:

- it does not support `since` param
- but it filters data based on `start date` param, here:
   https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-github/source_github/streams.py#L294